src/app/portlets/content-types/content-types-resolver.service.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@angular/core';
```

Comment:
I think is better named it something like: content-types.resolver.ts, it is not really a service

Reply:
I'm following the same pattern as with the guards as it is in the [angular documentation](https://angular.io/guide/router) they do like: ```src/app/crisis-center/crisis-detail-resolver.service.ts```